### ISSUE TITLE
I don't think this mod works with the GOG version

### ISSUE THREAD

**Issue by StentMan** [wofsauge/External-Item-Descriptions#277]
I have repentance and all 3 DLC installed, I tried using _[AB+ | Rep] External item descriptions_ with the game but no information shows up.
Also, I've experienced some crashed. Could be unrelated.

I used the latest version of the mod. According to the log file:
> Binding of Isaac: Repentance v1.06.J739

The screenshot shows how the mod menu looks (yes, that other mod doesn't seem to do its thing as well
![Untitled](https://user-images.githubusercontent.com/6360751/146639580-1b58c578-5c60-4579-836e-abe69a1af53b.png)
)

**Comment by wofsauge** [comment]
The most recent game version is :
> Binding of Isaac: Repentance v1.7.7a.0000
You seem to be using an older version of the game. Try to update it.

on the screenshot, it looks likeboth your mods are disabled, since the font color is gray. it needs to be the same color as the "Mods" text if its enabled. To enable it, select the mod and press space.

Lastly, when ingame, try to take a look at the debug console and write me what errors it displays. here is how to open it: https://bindingofisaacrebirth.fandom.com/wiki/Debug_Console

**Comment by StentMan** [comment]
Thanks for your answer ,wofsauge.
Yes, I disabled them b/c they didn't work and the game crashed. The "!!" looked strange to me, I hoped that'll hint at what's going on, maybe. That's why I posted this image.
As for the game version, that's the latest version available on GOG.

I enabled just the _External Item Descriptions_ mod and started a run, came across an item room, pressed F2 just in case, pulled up the debug console and took a screenshot:
![Untitled](https://user-images.githubusercontent.com/6360751/146650413-19a42a40-26d9-4ec3-8d8b-5e03e20a7fd1.png)

I hope this helps. I'll gladly provide additional information.
